bedrock/mozorg/templates/mozorg/home/includes/pocket-best-of.html

```diff
@@ -0,0 +1,19 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<aside class="pocket-feature mzp-l-content">
+  <div class="feature-wrapper">
+    <div class="image-wrapper">
+      <img loading="lazy" src="{{ static('img/home/2023/pocket-best-of.png') }}" alt="Pocket: Best of 2023">
+    </div>
+    <div class="feature-info">
+      <small>Featured product</small>
+      <h2>Discover the top articles of 2023</h2>
```

Comment:
This banner features hard-coded English strings. Either we should translate them, or restrict the banner to show for English locales only?